routes/api.router.js

```diff
@@ -1,8 +1,10 @@
 const apiRouter = require("express").Router();
 const topicsRouter = require("./topics.router");
+const articlesRouter = require('./articles.router')
 
 
-apiRouter.use("/topics", topicsRouter);
 
+apiRouter.use("/topics", topicsRouter);
```

Comment:
Really like that you've used routers to split out the endpoints into their own seperate sections